vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.cc

```diff
@@ -42,6 +62,30 @@ void Create::Start(ledger::CreateWalletCallback callback) {
       ledger::URL_METHOD::GET, on_req);
 }
 
+void Create::StartSafetyNetCallback(
+    int response_status_code,
+    const std::string& response,
+    const std::map<std::string, std::string>& headers,
+    ledger::CreateWalletCallback callback) {
+
+  ledger_->LogResponse(__func__, response_status_code, response, headers);
+
+  unsigned int statusCode;
+  std::string error;
+  bool hasResponseError = braveledger_bat_helper::getJSONResponse(
+    response, &statusCode, &error);
+  std::string message;
+  if (statusCode == SAFETYNET_ERROR_CODE || (hasResponseError &&
```

Comment:
`SAFETYNET_ERROR_CODE` should be `kSafetyNetErrorCode`, due to namespace issues Chromium changed the style where all new code should use `kFooBar` style for enums

Reply:
NONCRITICAL:
covered in: https://github.com/brave/brave-browser/issues/6200